bundles/org.openhab.binding.souliss/README.md

```diff
@@ -0,0 +1,352 @@
+|-- |-- | -- | -- | -- |--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|
+|souliss:t11|x|x|x|x||||
+|souliss:t12|x| |x|x|x||||
+|souliss:t13|||x|x||x|x||
+|souliss:t14|||x|x||||x|
+|souliss:t16|x|x|x|x|||||x|x|x|x||
+|souliss:t18|x||x|x|||||||||
+|souliss:t19|x|x|x|x||||||x|x|
+|souliss:t1A|||x|x|||||||||x|x|x|x|x|x|x|x|
+
+|Thing type / Channel | DateTime / lastStatusStored | Number / healthy |Rollershutter / rollershutter|(see down) / rollershutterstate|(see down) / mode|(see down) / fan|Switch / status|Number / setpoint|Switch / setAsMeasured|Switch / measured|Switch / statusAlarm|Switch / onOffAlarm|Switch / rearmAlarm|
+|-- |-- | --|- | -- | -- |--|--|--|--|--|--|--|--|
+|souliss:t21|x|x||x|
+|souliss:t22|x|x|x|x|
+|souliss:t31|x|x|||x|x|x|x|x|x|
+|souliss:t41|x|x|||||||||x|x|x|
+|souliss:t42|x|x|||||||||x||x|
+
+rollershutterstate = opening, closing, limSwitch_open , limSwitch_close, state_open, state_close, NoLimSwitch
+
+mode = COOLING_MODE, HEATING_MODE, POWEREDOFF_MODE
+
+fan = AUTO, HIGH, MEDIUM, LOW, FANOFF
+
+
+|Thing type / Channel| DateTime / lastStatusStored |Number / healthy|Number / value|
+|-- |-- | -- | --|
+|souliss:t51|x|x|x|
+|souliss:t52|x|x|x|
+|souliss:t53|x|x|x|
+|souliss:t54|x|x|x|
+|souliss:t55|x|x|x|
+|souliss:t56|x|x|x|
+|souliss:t57|x|x|x|
+|souliss:t58|x|x|x|
+
+|Thing type / Channel|  DateTime / lastStatusStored |Number / healthy|Number / value|
+|-- |-- | -- | --|
+|souliss:t61|x|x|x|
+|souliss:t62|x|x|x|
+|souliss:t63|x|x|x|
+|souliss:t64|x|x|x|
+|souliss:t65|x|x|x|
+|souliss:t66|x|x|x|
+|souliss:t67|x|x|x|
+|souliss:t68|x|x|x|
+|souliss:topic|x||x|
+
+### Parameters
+
+|Thing type| Parameters Name and Default Value| Description|
+|-- |-- |-- |
+|Gateway|gatewayIpAddress="" |Will be resolved by discovery if auto configured|
+|"|gatewayPortNumber=230 ||
+|"|preferredLocalPortNumber=0 |Default port is 0. It means that it is random|
+|"|pingInterval=30 |Interval in seconds to check for device presence|
+|"|subscriptionInterval=2|Interval in minutes to subcribe Souliss Gateway|
+|"|healthyInterval=35|Interval in seconds to send nodes healthy|
+|"|userIndex=70|Generally the default value work good. It must is different from other user interfaces (ex: SoulissApp)|
+|"|nodeIndex=120|Generally the default value work good. It must is different from other user interfaces (ex: SoulissApp)|
+|T11|sleep=5|Set sleep timer in cycles|
+|T12|||
+|T13|||
+|T14|||
+|T16|sleep=5|Set sleep timer in cycles|
+|T19|sleep=5|Set sleep timer in cycles|
+|T1A|||
+|T21|||
+|T22|||
+|T31|||
+|T4x|||
+|T5x|||
+|T6x|||
+
+
+## Manual Things Configuration
+
+If after discovery your thing is not listed you can add it manually.
+You have to choice it from disponible items. 
```

Comment:
```suggestion
You have to choose it from the available things. 
```

We try to keep generic how to use openHAB out of a bindings readme. This way if the UI changes we do not have a heaps of documents to update, as the information on how to add manual things is already covered once in the getting started guide. I feel some of this section can be removed to make it shorter for this reason.